ui/app/com/gu/recipeasy/views/app.scala.html

```diff
@@ -1,8 +1,14 @@
 @import com.gu.recipeasy.models._
 @(title: String, userFirstname:String, progressBarPercentage: Double, generalStats: Map[GeneralStatisticsPoint,Long], userStats: Map[PersonalStatisticsPoint, String])
 
+@decimalTo3Digit(number: Double) = @{
+    ((number)*1000).toInt.toDouble/1000
```

Comment:
If not, there will be lots of Java classes that have this functionality such as `https://docs.oracle.com/javase/7/docs/api/java/text/NumberFormat.html`